easybuild/easyconfigs/i/ipyparallel/ipyparallel-6.2.2-foss-2018a-Python-3.6.4.eb

```diff
@@ -0,0 +1,39 @@
+easyblock = 'PythonPackage'
+
+name = 'ipyparallel'
+version = '6.2.2'
+python = "Python"
+pythonversion = "3.6.4"
+versionsuffix = "-%s-%s" % (python, pythonversion)
+
+homepage = 'https://ipyparallel.readthedocs.io'
+description = """ipyparallel is a Python package and collection
+ of CLI scripts for controlling clusters for Jupyter"""
+
+toolchain = {'name': 'foss', 'version': '2018a'}
+
+source_urls = ['https://pypi.python.org/packages/source/p/ipyparallel']
+sources = [SOURCELOWER_TAR_GZ]
+checksums = ['02b225966d5c20f12b1fba0b6b10aa5d352a6b492e075f137ff0ff6e95b9358e']
+
+dependencies = [
+    (python, pythonversion),
+    ('IPython', '6.4.0', versionsuffix),
+]
+
+use_pip = True
+
+exts_download_dep_fail = True
+
+full_sanity_check = True
+
+modextrapaths = {'PYTHONPATH': ['lib/python%(pyshortver)s/site-packages']}
+
+sanity_check_paths = {
+    'files': [],
+    'dirs': ['lib/python%(pyshortver)s/site-packages'],
+}
+
+sanity_check_commands = ["python -c 'import ipyparallel'"]
```

Comment:
@SimonPinches No need for this, this is done automatically by `PythonPackage` (the check is basically done twice now, no need for that).